Maskbook/Scene/Wallet/Balance/View/BalanceTokenCell.swift

```diff
@@ -75,8 +75,8 @@ class BalanceTokenCell: UITableViewCell {
         containerView.addSubview(tokenBalanceLabel)
         
         NSLayoutConstraint.activate([
-            containerView.leadingAnchor.constraint(equalTo: contentView.readableContentGuide.leadingAnchor),
-            containerView.trailingAnchor.constraint(equalTo: contentView.readableContentGuide.trailingAnchor),
+            containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: LayoutConstraints.leading),
+            containerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -LayoutConstraints.trailing),
```

Comment:
Why changes to fixed padding value? This would break the appearance on iPad

Reply:
The UI designer told me to adjust the horizontal margin to 20pt, where it is not aligned with the rest. 
Or do I adjust all the places where the margins are 20 to `readableContentGuide`?

Reply:
Plz adjust the padding with `readableContentGuide`